.env.example

```diff
@@ -0,0 +1,6 @@
+export ETH_RPC_URL=https://eth-mainnet.alchemyapi.io/v2/yourAlchemyApiKey
+export FORK_BLOCK=13724056
+export GAS_LIMIT=30000000
+
+# skip type-checking to avoid its perfomance impact
+export TS_NODE_TRANSPILE_ONLY=1
```

Comment:
Should have newline at end of file. (I `cat` files like this!)

Reply:
Done in f7f3b78